sdk/core/core-rest-pipeline/src/policies/proxyPolicy-react-native.mts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/*
+ * NOTE: When moving this file, please update "browser" section in package.json
+ */
+
+export const proxyPolicyName = "proxyPolicy";
```

Comment:
This file appears to be a copy of `proxyPolicy.common.ts`, shouldn't it re-export from that file instead like the browser one does?

Reply:
Fixed to be the same export as the browser.